XamarinCommunityToolkit/Behaviors/MaskedBehavior.shared.cs

```diff
@@ -0,0 +1,88 @@
+﻿using System;
+using System.Collections.Generic;
+using Xamarin.Forms;
+
+namespace Microsoft.Toolkit.Xamarin.Forms.Behaviors
+{
+    public class MaskedBehavior : BaseBehavior
+    {
+        IDictionary<int, char> positions;
+
+        public static readonly BindableProperty MaskProperty =
+            BindableProperty.Create(nameof(Mask), typeof(string), typeof(MaskedBehavior), propertyChanged: OnMaskPropertyChanged);
+
+        public static readonly BindableProperty UnMaskedCharacterProperty =
+            BindableProperty.Create(nameof(UnMaskedCharacter), typeof(char), typeof(MaskedBehavior), 'X', propertyChanged: OnUnMaskedCharacterPropertyChanged);
+
+        public string Mask
+        {
+            get => (string)GetValue(MaskProperty);
+            set => SetValue(MaskProperty, value);
+        }
+        public char UnMaskedCharacter
+        {
+            get => (char)GetValue(UnMaskedCharacterProperty);
+            set => SetValue(UnMaskedCharacterProperty, value);
+        }
+
+        protected override void OnAttachedTo(View bindable)
+        {
+            ((InputView)bindable).TextChanged += OnEntryTextChanged;
+            base.OnAttachedTo(bindable);
+        }
+        protected override void OnDetachingFrom(View bindable)
+        {
+            ((InputView)bindable).TextChanged -= OnEntryTextChanged;
+            base.OnDetachingFrom(bindable);
+        }
```

Comment:
Instead of doing this, just override OnViewPropertyChanged method and check if property name is Entry.TextProperty.PropertyName

Reply:
@AndreiMisiukevich can do, but id also need to do that for the editor and possible the search bar as well. It would also mean anyone who has built their own control that inherits input view wouldn't able to use it (thinking packages like Synctusion, Grail etc)

Reply:
That's just a string check :) Entry.TextProperty.PropertyName == "Text") You can do it as InputView.TextProperty.PropertyName

Reply:
@AndreiMisiukevich oh ok, would i need to change something in the `OnEntryTextChanged` method? - 
```csharp
var inputView = sender as InputView;
var text = inputView.Text;
```

Reply:
var text = ((InputView)View).Text;